src/activities/binary_bulb/BinaryBulb.qml

```diff
@@ -0,0 +1,169 @@
+/* GCompris - binary_bulb.qml
+ *
+ * Copyright (C) 2018 Rajat Asthana <rajatasthana4@gmail.com>
+ *
+ * Authors:
+ *   RAJAT ASTHANA <rajatasthana4@gmail.com> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.6
+import GCompris 1.0
+
+import "../../core"
+import "binary_bulb.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: background
+        anchors.fill: parent
+        source: "resource/background.svg"
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            property alias bulbs: bulbs
+            property int sum: 0
+            property int num: 0
+            property int numberOfBulbs: 0
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        DialogHelp {
+            id: dialogHelp
+            onClose: home()
+        }
+
+        Bar {
+            id: bar
+            content: BarEnumContent { value: help | home | level}
+            onHelpClicked: {
+                displayDialog(dialogHelp)
+            }
+            onPreviousLevelClicked: Activity.previousLevel()
+            onNextLevelClicked: Activity.nextLevel()
+            onHomeClicked: activity.home()
+        }
+
+        Bonus {
```

Comment:
Can you place this and the above two elements at the bottom of the file? See other activities for example.

Reply:
yeah sure :)